### ISSUE TITLE
I suggest you to use BEM methodology

### ISSUE THREAD

**Issue by BumbuKhan** [hasan-li/develolper#4]
You can read more about it here http://getbem.com/introduction/

**Comment by hasan-li** [comment]
Is it still relevant?

**Comment by device999** [comment]
Is it like bootstrapp?

**Comment by BumbuKhan** [comment]
@device999 no, this it nothing alike than Bootstrap! BEM is a methodology, a bunch of conventions about writing code, naming classes, splitting whole UI into reusable components and so forth... 
I find it quite useful since it helps developers to be on the same wave and write code in the same way.
I strongly recommend you to take a look at it's documentation (link is provided above).

@hasan-li we're a team and should decide together whether it's relevant or not.

**Comment by BumbuKhan** [comment]
Here you can see 3 various naming styles:

![img_1834](https://cloud.githubusercontent.com/assets/19274459/24836658/5817699e-1d32-11e7-8464-8836b4e288e5.PNG)
![img_1835](https://cloud.githubusercontent.com/assets/19274459/24836659/585c6198-1d32-11e7-9963-2e3c3dab8470.PNG)

**Comment by hasan-li** [comment]
Then we go with it!
